src/iam/login/login.service.spec.ts

```diff
@@ -94,26 +105,19 @@ describe('LoginService', () => {
       expect(await loginService.login(loginDto)).toEqual(userLogin);
     });
 
-    it('should return an exception if login fails', async () => {
-      jest
-        .spyOn(loginService, 'login')
-        .mockRejectedValue(new UnauthorizedException('User does not exists'));
-
-      await expect(loginService.login(loginDto)).rejects.toThrow(
-        new UnauthorizedException('User does not exists'),
-      );
-    });
```

Comment:
This mocks the method you're testing. It adds no value to the suite

Reply:
Yes indeed I will remove them soon